tools/cloud-workstations/Dockerfile

```diff
@@ -40,3 +40,16 @@ RUN pip install --no-cache-dir --upgrade pip && \
     pip install --no-cache-dir -r https://raw.githubusercontent.com/SchedMD/slurm-gcp/master/scripts/requirements.txt && \
     pip install --no-cache-dir -r https://raw.githubusercontent.com/GoogleCloudPlatform/hpc-toolkit/main/tools/cloud-build/requirements.txt && \
     rm -rf ~/.cache/pip/*
+
+ARG HPC_TOOL_URL=https://raw.githubusercontent.com/GoogleCloudPlatform/hpc-toolkit/main/tools/cloud-workstations
```

Comment:
@cboneti I'm not sure which URL/branch to use here.  In the example above, the main branch is used, but until this is merged from develop into main, it will be incorrect. (updated as I put this comment on the wrong line before)

Reply:
This is OBE with my new push.